platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/migratable/ProfileMigratable.java

```diff
@@ -143,7 +135,8 @@ private boolean restore(MigrationReport report, Optional<InputStream> ois) throw
         JsonUtils.fromJson(IOUtils.toString(is, StandardCharsets.UTF_8), JsonProfile.class);
 
     for (int i = 1; i < ProfileMigratable.RETRY_COUNT; i++) {
```

Comment:
❓ Should this condition be i <= ProfileMigratable.RETRY_COUNT? From the log message below, it seems like it should be.

Reply:
It should be  `<` since there is one last attempt after the loop for a total of RETRY_COUNT. It was `<=` before Tracy had me move the last attempt out of the loop.